research/report-from-work-done-on-signature-verification-issue.md

```diff
@@ -0,0 +1,58 @@
+# Report From Work Done On Signature Verification Issue
+
+> Problem is described within [Vitro Oracle PoC Release Notes](https://github.com/VitroTech/release-notes/blob/2fb0391512e492c5d875828b1b4b90d98557e8b4/2020/oracle-poc-2020-07-22/release-notes.md).
+
+### Steps Done
+
+ 1. Research on core concepts related to the issue. As it's a more complex
```

Comment:
@rg3mdeb maybe we shouldn't write timeframes on each task + do not write in the first-person in the docs/reports

Reply:
Won't happen again. Fixed